server/routes/dashboard.js

```diff
@@ -25,14 +25,16 @@ router.post(['/search/reference', '/api/search/reference'],
         const logger = getLogger(req.requestId);
         try {
             const caseRef = req.form.data['case-reference'].toUpperCase();
+
```

Comment:
if it were me I'd remove this new line to keep the "caseRef declaration stuff" in one block. But that's all personal preference.